config/config.properties

```diff
@@ -299,6 +299,15 @@ metrics.statistics.enable=false
 # InfluxDB retention policy (e.g. 2h, 52w).
 #metrics.influxDB.retention=
 
+# Whether to enable the Prometheus listener.
```

Comment:
Wondering if we need to update https://github.com/pravega/pravega/blob/master/documentation/src/docs/metrics.md#metrics-service-provider--interface-statsprovider to mention that we also support Prometheus now, right?

Reply:
done, just added it to the existing phrase in 0489f23